maintenance_collaboration.Rmd

```diff
@@ -70,7 +70,7 @@ Please do not list editors as contributors. Your participation in and contributi
 
 ### Welcoming collaborators to rOpenSci
 
-If you give someone write permissions to the repository, please contact one of [the editors](#associateditors) or [Stefanie Butland](https://github.com/stefaniebutland) so that this new contributor might
+If you give someone write permissions to the repository, please contact one of [the editors](#associateditors) or rOpenSci community manager [Stefanie Butland](https://github.com/stefaniebutland) so that this new contributor might
```

Comment:
I have not previously been involved in inviting people to our GitHub org and prefer it that way.
Edit is ok otherwise

Reply:
So should I remove your name from here? I'm not even sure I have been contacted for that either.

Reply:
yes please.

but ok to tag me to invite ppl to slack. I prefer to be the one to do that